proxy/src/main/scala/tech/beshu/ror/proxy/es/services/EsRestClientBasedRorClusterService.scala

```diff
@@ -61,6 +66,20 @@ class EsRestClientBasedRorClusterService(client: RestHighLevelClientAdapter)
     allTemplates.find(_.name === name)
   }
 
+  //TODO
+  override def verifyDocumentAccessibility(document: Document,
+                                           filter: domain.Filter,
+                                           id: RequestContext.Id): Task[domain.DocumentAccessibility] = {
+    Task.now(Inaccessible)
```

Comment:
could you please implement new methods also for proxy? Since https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/pull/593 we should have to think and support also proxy.